tests/test_hintfontsmm.py

```diff
@@ -0,0 +1,28 @@
+import glob
+import pytest
+
+from fontTools.misc.xmlWriter import XMLWriter
+from fontTools.ttLib import TTFont
+from psautohint.autohint import ACOptions, hintFiles
+
+from .differ import main as differ
+
+
+class Options(ACOptions):
+    def __init__(self, reference, inpaths, outpaths):
+        super(Options, self).__init__()
+        self.inputPaths = inpaths
+        self.outputPaths = outpaths
+        self.reference_font = reference
+        self.hintAll = True
+        self.verbose = False
+
+
+@pytest.mark.parametrize("base", glob.glob("data/*/*Masters"))
+def test_mmufo(base):
+    paths = sorted(glob.glob(base + "/*.ufo"))
+    options = Options(paths[0], paths[1:], [p + ".out" for p in paths[1:]])
```

Comment:
better to define `reference`, `inpaths` and `outpaths` outside, I think

Reply:
Naming variables is hard, I struggled with finding appropriate names so I avoided them altogether :) Using your suggestion now.